src/repl.rs

```diff
@@ -1,7 +1,13 @@
 use std::error::Error as StdError;
+use std::fmt;
+use std::sync::Mutex;
+
+use rustyline::error::ReadlineError;
```

Comment:
Wasn't aware you could put extern crate in other modules than the root actually.